### PR TITLE
5 sprint 2issues 3sign in and sign up

### LINKED ISSUE #5: sprint-2/issues-3#Sign In and Sign Up
## REQUIREMENTS / DESCRIPTION

> Create the Sign In and Sign Up screens, which is a screen to be able to register as well as login when you have a registered account you will be able to register manually with simple information as well as automatically with facebook or google.

## DESIGN
image
> image
image

### PR DESCRIPTION
**Pull request by FelixAva**
# Description

Create Sign In and Sign Up screens. 

# Design
image
image

### CODE REVIEW

File: src/assets/img/hide.png
Comment (VISOSO2403) [inline src/assets/img/hide.png:1]: delete this image

File: src/assets/img/visible.png
Comment (VISOSO2403) [inline src/assets/img/visible.png:1]: delete this image

**Review by Jherrera11 — APPROVED** [review]
**LGTM !!!**
image

**Review by VISOSO2403 — CHANGES_REQUESTED** [review]
resolve request changes

**Review by Antonio3veces3 — CHANGES_REQUESTED** [review]
Resolve conflicts before merge!

**Review by VISOSO2403 — APPROVED** [review]
LGTM

**Pull request merged**